guides/common/modules/proc_synchronizing-templates-with-a-local-directory.adoc

```diff
@@ -0,0 +1,49 @@
+[id="Synchronizing_Templates_with_a_Local_Directory_{context}"]
```

Comment:
```suggestion
[id="Synchronizing_Templates_with_a_Local_Directory_{context}"]
```
Choose from either Sentence case or lower case.

Reply:
Actually, Camel Case is very common. You can verify with `git grep -E 'id="[A-Z]'`

Reply:
Right, but here `proc_synchronizing-templates-with-an-existing-repository.adoc` you have used the smallcase. It should be the same for all?

Or, is it flexible?

Reply:
I believe it is flexible. Smallcase is generated by `newdoc` while some people who create modules manually use Camel Case. So far both styles are accepted.

Reply:
:+1: